platforms/winit/examples/simple.rs

```diff
@@ -83,13 +83,17 @@ impl State {
         let root = self.build_root();
         let button_1 = build_button(BUTTON_1_ID, "Button 1", &mut self.node_classes);
         let button_2 = build_button(BUTTON_2_ID, "Button 2", &mut self.node_classes);
+        let mut tree = Tree::new(WINDOW_ID);
+        tree.app_name = Some(env!("CARGO_BIN_NAME").to_string());
+        tree.toolkit_name = Some(String::from("winit"));
+        tree.toolkit_version = Some(String::from("0.28"));
```

Comment:
We'll now have to manually update this everytime we bump winit's version. winit isn't even the UI toolkit here, just a windowing library. I'm now pretty sure it's better to remove toolkit name and version from this example and the SDL one.